### PR TITLE
Fix gorm remove PreparedSQL field

### PR DESCRIPTION
**Pull request by smalls0098**
- [ ] Do only one thing
- [ ] Non breaking API changes
- [ ] Tested

### What did this pull request do?

# gorm.io/plugin/dbresolver
/Users/smalls/go/pkg/mod/gorm.io/plugin/dbresolver@v1.5.2/dbresolver.go:147:5: unknown field PreparedSQL in struct literal of type gorm.PreparedStmtDB

### CODE REVIEW

**Comment by smalls0098** [comment]
> 请合并 !!!! :(

go.mod 添加一行
替换 gorm.io/plugin/dbresolver v1.5.2 => github.com/smalls0098/dbresolver v0.0.0-20240909115957-b928cd627cd8
临时解决，等待修复
image

**Comment by deefdragon** [comment]
@jinzhu This is breaking builds for several people. A merge & version update would be appreciated.

File: dbresolver.go [hunk 1]
```diff
@@ -144,7 +144,7 @@ func (dr *DBResolver) convertToConnPool(dialectors []gorm.Dialector) (connPools
 				ConnPool:    db.Config.ConnPool,
 				Stmts:       map[string]*gorm.Stmt{},
 				Mux:         &sync.RWMutex{},
-				PreparedSQL: make([]string, 0, 100),
+				//PreparedSQL: make([]string, 0, 100), // gorm@v1.25.12 remove
```
Comment (a631807682) [inline dbresolver.go:147]: No need to comment, just delete it.

**Comment by deefdragon** [comment]
@a631807682 is there any chance you can add a new tag now that this has been merged? v1.5.3?

**Comment by sunerpy** [comment]
@a631807682 please add a new tag

**Comment by a631807682** [comment]
@deefdragon @sunerpy use v1.5.3

**Pull request merged**